nx-X11/programs/Xserver/GL/include/GL/xf86glx.h

```diff
@@ -1,42 +0,0 @@
-/**************************************************************************
```

Comment:
Within this commit, we could probably drop `-Iinclude` from `nx-X11/programs/Xserver/GL/Imakefile`, as this directory is gone now.